riak_test/yz_rt.erl

```diff
@@ -615,3 +615,6 @@ load_intercept_code(Node) ->
                               "*.erl"]),
     rt_intercept:load_code(Node, [CodePath]).
 
+-spec rpc_call_cluster([node()], Module :: atom(), Function :: atom(), [term()]) -> [term()].
+rpc_call_cluster(Cluster, Module, Function, Args) ->
+    [rpc:call(Node, Module, Function, Args) || Node <- Cluster].
```

Comment:
Is this any different from Erlang's built-in `rpc:multicall/4`?

http://erlang.org/documentation/doc-5.1/lib/kernel-2.7/doc/html/rpc.html#multicall%4